app/models/Scholarship.php

```diff
@@ -99,7 +99,7 @@ public static function getScholarshipLabels($id)
    */
   public static function getPastScholarship($id)
   {
-      $path = Cache::remember('scholarships', 120, function () use ($id) {
+      $path = Cache::remember('scholarships.'.$id, 120, function () use ($id) {
```

Comment:
This is really nitpicky but I think we usually use a `:` as a separator for keys in cache

Reply:
Ahh maybe it should be that. Elsewhere in this app `.` is always used so I'm going to keep it consistent for now.